mozilla_bitbar_devicepool/lambdatest/user_script/setup_script.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+set -e
+set -x
+
+starting_dir=$(pwd)
+echo "starting_dir: $starting_dir"
+
+sudo apt update -y
+sudo apt-get install gettext-base
+sudo apt-get install libgtk-3-0 -y
+
+pip install zstandard
+
+# aje 3/17/25: not needed per LT.
+#   - was interfering with their monitoring and made the main testRunCommand not work.
+#
+# install adb
+# sudo rm -f /usr/bin/adb
+# sudo rm -f /home/ltuser/adb-original/adb
+# sudo rm -rf /usr/lib/android-sdk/platform-tools
+# sudo mkdir -p /usr/local/android-sdk
+# # if /usr/local/android-sdk/ doesn't exist, we need to install it
+# if [ ! -d "/usr/local/android-sdk" ]; then
+#     cd /usr/local/android-sdk/
+#     sudo curl -OL https://dl.google.com/android/repository/platform-tools-latest-linux.zip
+#     sudo unzip platform-tools-latest-linux.zip
+#     sudo rm -f platform-tools-latest-linux.zip
+#     sudo ln -s /usr/local/android-sdk/platform-tools/adb /usr/bin/adb
+#     sudo cp -r /usr/local/android-sdk/platform-tools /usr/lib/android-sdk/
+# fi
+
+rm -Rf taskcluster/
+
+# setup taskcluster client in home/ltuser:
+# assume taskcluster/* was copied over
+# TODO: either clone repo, or build package for single download;  a lot of this is added via dockerfile
+cd /home/ltuser/
+mkdir taskcluster
+cd taskcluster
+wget -O generic-worker https://github.com/taskcluster/taskcluster/releases/download/v80.0.0/generic-worker-insecure-linux-amd64
+wget -O livelog https://github.com/taskcluster/taskcluster/releases/download/v80.0.0/livelog-linux-amd64
+wget -O taskcluster-proxy https://github.com/taskcluster/taskcluster/releases/download/v80.0.0/taskcluster-proxy-linux-amd64
+wget -O start-worker https://github.com/taskcluster/taskcluster/releases/download/v80.0.0/start-worker-linux-amd64
```

Comment:
note: this is pinned to v80.0.0, could we either:
 * check in to a repo what we want?
 * put the version into a variable?

Reply:
Version now in a variable.